src/components/AddEventToGroupForm/AddEventToGroupForm.tsx

```diff
@@ -0,0 +1,179 @@
+import { Activity } from "@components/ActivitiesList/ActivitiesList";
+import {
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  TextField,
+} from "@mui/material";
+import axios, { AxiosError } from "axios";
+import React, { useEffect, useState } from "react";
+
+import EventStatus from "./EventStatus";
+
+const activityEndpoint = `${import.meta.env.VITE_BACKEND_URL}activities`;
+const eventEndpoint = `${import.meta.env.VITE_BACKEND_URL}events`;
+
+const AddEventToGroupForm: React.FC = () => {
+  const [eventName, setEventName] = useState<string>("");
+  const [eventDateTime, setEventDateTime] = useState<string>(
+    new Date().toISOString()
+  );
+  const [eventDescription, setEventDescription] = useState<string>("");
+  const [eventActivities, setEventActivities] = useState<Activity[]>([]);
+  const [selectedActivities, setSelectedActivities] = useState<Activity[]>([]);
+  const [formSubmitted, setFormSubmitted] = useState<{
+    isSuccess: boolean;
+    message: string;
+  } | null>(null);
+
+  const handleDateTimeChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const selectedDateTime = e.currentTarget.value;
+    const isoDateTime = new Date(selectedDateTime).toISOString();
+    setEventDateTime(isoDateTime);
+  };
+
+  const handleCreateEvent = async () => {
+    try {
+      const response = await axios.post(
+        eventEndpoint,
+        {
+          name: eventName,
+          groupID: "657d0f422cb49ec9046b3b8c",
```

Comment:
@catbus00, heads up that @marice-romero is working on implementing a link from the `/group/id` page to this form...we should be able to pass a group id that way so that we can populate that here. Very clever of you to hardcode what was missing just to get everything else working though! Great thinking! 💯